spec/changelog_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'fastlane_core/update_checker/changelog'
+
+release_json_file = "spec/responses/fastlane_releases.json"
+
+RSpec.configure do |config|
+  config.before(:each) do
+    stub_request(:get, FastlaneCore::Changelog.generate_gem_releases_url("fastlane")).
+      with(:headers => {'Host'=>'api.github.com:443', 'User-Agent'=>'excon/0.45.4'}).
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.